### PR TITLE
Feature/occ 46 provisional implementation of terms of use

### PR DESCRIPTION
**Pull request by MacGyver-NRW**
### 1. Why is this change necessary?
Write down new terms of use and integrate a new responsive framework to OC

### 2. What does this change do, exactly?
New template /terms.tpl
Update menu on correct link

### 3. Describe each step to reproduce the issue or behaviour.
-

### 4. Please link to the relevant issues (if any).
OCC-46_provisional_implementation_of_terms_of_use
OCC-47_Implement_MDBootstrap_Framework 

### 5. Checklist

- [ ] I have written tests and verified that they fail without my change
- [ ] I have squashed any insignificant commits
- [ ] This change has comments for package types, values, functions, and non-obvious lines of code

### CODE REVIEW

File: htdocs/index.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
 $tpl->menuitem = MNU_START;
 
 $tpl->caching = true;
-$tpl->cache_lifetime = 300;
+$tpl->cache_lifetime = 60;
```
Comment (nlubisch) [inline htdocs/index.php:19]: Why change the global cache lifetime?

File: htdocs/index.php [hunk 2]
```diff
@@ -30,6 +30,15 @@
         $tpl->assign('message', $translate->t('You can find everything you need to go Geocaching ...', '', '', 0));
     }
 
+    // get HQ Message from DB
+    $aGetHqMessage = $connection->fetchAssoc(
```
Comment (nlubisch) [inline htdocs/index.php:34]: Can't we create a entity, repository and service for this und call it here?

**Review by nlubisch — CHANGES_REQUESTED** [review]

File: htdocs/index.php (hunk 1 above)
Comment (MacGyver-NRW) [reply htdocs/index.php:19]: That's too much for an index page. We update a lot of data here. A caching to 60 seconds is sufficient for a highly frequented page. New content will only appear after 5 minutes. The newly installed core_hq_message is also time-controlled and would stay there too long.

File: htdocs/index.php (hunk 2 above)
Comment (MacGyver-NRW) [reply htdocs/index.php:34]: At first I kept to the outdated code format in order not to introduce a new format in the old code.

File: htdocs/app/Migrations/Version20190211224434.php [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20190211224434 extends AbstractMigration
+{
+    public function up(Schema $schema)
+    : void {
+        $this->addSql('UPDATE `sys_menu` SET `href` = `articles.php?page=terms` WHERE id_string = `MNU_START_TOS`;');
```
Comment (nlubisch) [inline htdocs/app/Migrations/Version20190211224434.php]: MNU_START_TOS is surrounded by backticks and must be surrounded by single quotes in order to pass travis-ci

**Review by nlubisch — CHANGES_REQUESTED** [review]

**Review by teiling88 — APPROVED** [review]

**Review by teiling88 — APPROVED** [review]

**Pull request merged**